src/services/__tests__/tourettes.test.js

```diff
@@ -22,7 +22,17 @@ describe('tourettes', () => {
       it('replies with at least three items from the profanities list', () => {
         const reply = sayTourettes(inputText);
         expect(reply).toNotBe(undefined);
-        expect(reply.split(' ').length).toBeGreaterThan(3);
+        expect(reply.split(' ').length).toBeGreaterThan(2);
+      });
+    });
+
+    describe('when input text is integer', () => {
+      const inputText = '25';
+
+      it('replies with a specified amount of profanities', () => {
+        const reply = sayTourettes(inputText);
+        expect(reply).toNotBe(undefined);
+        expect(reply.split(' ').length).toBeGreaterThan(24);
```

Comment:
It would be nicer to make use of the integer inside inputText for the check if greater than here.